commit cadda07e052373f8fb6894984ce332105393ec9a
Author: Richard Hughes <richard@hughsie.com>
Date:   Wed Apr 9 02:11:02 2008 +0100

    hide the icon when we are updating multiple packages also

diff --git a/src/gpk-notify.c b/src/gpk-notify.c
index 1d184cd..e619f34 100644
--- a/src/gpk-notify.c
+++ b/src/gpk-notify.c
@@ -993,7 +993,8 @@ gpk_notify_task_list_changed_cb (PkTaskList *tlist, GpkNotify *notify)
 {
 	g_return_if_fail (GPK_IS_NOTIFY (notify));
 	/* hide icon if we are updating */
-	if (pk_task_list_contains_role (tlist, PK_ROLE_ENUM_UPDATE_SYSTEM)) {
+	if (pk_task_list_contains_role (tlist, PK_ROLE_ENUM_UPDATE_SYSTEM) ||
+	    pk_task_list_contains_role (tlist, PK_ROLE_ENUM_UPDATE_PACKAGES)) {
 		gpk_smart_icon_set_icon_name (notify->priv->sicon, NULL);
 	}
 }
